src/components/dropdownSearch/DropdownSearch.tsx

```diff
@@ -326,10 +347,27 @@ export class DropdownSearch extends React.Component<IDropdownSearchProps, {}> {
     }
   }
 
+  handleOnKeyDownFilterBox(e: React.KeyboardEvent<HTMLInputElement>) {
```

Comment:
Should probably be private (handleOnKeyDownDropdownButton too)

Reply:
refactored methods accessibility in the whole file